packages/npm/todo/package.json

```diff
@@ -4,7 +4,11 @@
   "type": "module",
   "scripts": {
     "prepack": "node ./scripts/build.js",
-    "pretest": "tsc"
+    "pretest": "node ./scripts/build.js",
```

Comment:
adds a bunch of scripts. The pretest is interesting. It will run before tests are run with npm test. The build script is used by the vscode configuration. This script is run before running tests from vscode.